modules/statics/src/networks.ts

```diff
@@ -297,6 +297,13 @@ class BitcoinPublicSignet extends Testnet implements UtxoNetwork {
   explorerUrl = 'https://mempool.space/signet/tx/';
 }
 
+class BitcoinBitGoSignet extends Testnet implements UtxoNetwork {
+  name = 'BitcoinBitGoSignet';
+  family = CoinFamily.BTC;
+  utxolibName = 'bitcoinBitGoSignet';
+  explorerUrl = 'https://mempool.space/notanetwork/tx/';
```

Comment:
So we don't have a explorer for BitGo Signet?

Reply:
Not right now